certbot/compat/os.py

```diff
@@ -29,3 +29,15 @@
 
 # Clean all remaining importables that are not from the core os module.
 del ourselves, std_os, std_sys
+
+
+# The os.open function on Windows will have the same effect than a bare os.chown towards the given
```

Comment:
What does this mean? What effect, and on what? Which given mode? What does a "bare" os.chown mean? Where are those flaws described?

Reply:
Here we are on a side effect of splitting the logic in several PRs. This comment is written using a lot of reference to `os.chmod`, that is implemented in #6964.

So it should be read as it will be once the logic is merged: first the comment on `os.chown` gives the context about limitation of file modes in Windows, then this comment and the function `os.open` is written just after, reusing this context for the reader.